templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -69,9 +69,17 @@ contents:
           echo "error finding static-pod-resources"
           exit 1
       fi
+
+      # tar up the static kube resources, with the path relative to CONFIG_FILE_DIR
       tar -cpf $BACKUP_TAR_FILE -C ${CONFIG_FILE_DIR} ${LATEST_STATIC_POD_DIR#$CONFIG_FILE_DIR/}
     }
 
+    append_snapshot_to_tar_and_gzip() {
```

Comment:
question: if this function fails do you feel we will provide enough information to debug?

Reply:
Well. If this function fails, tar or gzip should report error, which should be available to the end user. Then, the trap removes whatever files we backed up. (Also the set +x debug tells us we came this far and failed but end user doesn't run with +x). I am not sure adding more error checking here is any useful.

Reply:
OK that makes sense, thanks